endpoints/road/all.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-prototype-builtins */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-unneeded-ternary */
+
+const request = require('request')
+const xml2js = require('xml2js')
+const h = require('apis-helpers')
+const app = require('../../server')
+
+const parseString = xml2js.parseString
+// return callback(null, JSON.parse(data))
+const parseFeed = function (callback, data) {
+  parseString(data, { explicitRoot: false }, (err, result) => {
+    if (err) return callback(new Error(`Parsing of XML failed. Title ${err}`))
+    const roads = []
+
+    for (let i = 0; i < result.Faerd.length; ++i) {
+      const Road = result.Faerd[i]
+      roads.push({
+        routeId: Road.IdLeid[0].length > 0 ? Road.IdLeid[0] : null, // Can be null
```

Comment:
I think you could write this as `Road.IdLeid[0] || null`. Same goes for the other lines that have similar logic.

Reply:
unfortunately not :/

that gives me 
```
segmentSignal: {
$: {
i:nil: "true"
}
},
```